app/controllers/api/api_controller.rb

```diff
@@ -6,20 +6,25 @@ def options
 
     def index
       res = {
-        :name         => ApiConfig.base.name,
-        :description  => ApiConfig.base.description,
-        :version      => ManageIQ::Api::VERSION,
-        :versions     => entrypoint_versions,
-        :settings     => user_settings,
-        :identity     => auth_identity,
-        :server_info  => server_info,
-        :product_info => product_info
+        :name          => ApiConfig.base.name,
+        :description   => ApiConfig.base.description,
+        :version       => ManageIQ::Api::VERSION,
+        :versions      => entrypoint_versions,
+        :settings      => user_settings,
+        :identity      => auth_identity,
+        :server_info   => server_info,
+        :product_info  => product_info_data,
+        :branding_info => branding_info
```

Comment:
Do we still need this if it's also nested under `product_info`?

Reply:
Well, we need to be able to access this without a login, so the SUI login screen can consume it.

Reply:
Doesn't https://github.com/ManageIQ/manageiq-api/pull/438/files#diff-b4f92b4e361ffab4fbb91c68eb3f77bbR23 remove the requirement for being logged in?

Reply:
No, the `/api` route isn't available without login as it contains the `identity` part that needs a logged in user.

Reply:
> Do we still need this if it's also nested under product_info?

What I meant was that we'll have this, right?
```json
get api_entrypoint_url

{
  "name" : "ManageIQ",
  "description" : "words",
  ...,
  "product_info" : {
    ...,
    "branding_info" : {...}
  },
  "branding_info" : {...}
}
```

Reply:
Oooh, you're right :roll_eyes: I don't know why I left that line there :cry: